pkg/daemon/containerrecreate/crr_daemon_util.go

```diff
@@ -114,7 +118,9 @@ func getCurrentCRRContainersRecreateStates(
 				syncContainerStatus.Ready {
 				currentState.Phase = appsv1alpha1.ContainerRecreateRequestSucceeded
 			}
-
```

Comment:
it is not necessary to force kill container if the container had already been sucessfully recreated